vllm/attention/backends/torch_sdpa.py

```diff
@@ -13,9 +13,12 @@
 
 if is_cpu():
     try:
+        from vllm._ipex_ops import ipex_ops
```

Comment:
Shouldn't we add IPEX to the CPU's dependency?

Reply:
We will add ipex dependency when its API is stable.
cc @zhouyuan @bigPYJ1151

Reply:
Sounds good.